### PR TITLE
1.6

### PR DESCRIPTION
**Pull request by pascalfree**
upgrade to juntagrico 1.6.0

### CODE REVIEW

File: README.md
Comment (samuel-iseli) [inline README.md:1]: why has the `juntagrico_billing` app to be above `juntagrico`? This wasn't a requirement up to now (in my local `settings.py` it is below `juntagrico`).

File: juntagrico_billing/admin/__init__.py [hunk 1]
```diff
@@ -23,6 +27,11 @@ class SubscriptionTypeAccountInline(admin.TabularInline):
     extra = 0
 
 
+# add inlines on juntagrico models
+addons.config.register_model_inline(Member, MemberAccountInline)
```
Comment (samuel-iseli) [inline juntagrico_billing/admin/__init__.py:31]: Why do we need to register juntagrico admin stuff here? Shouldn't this get registered in `juntagrico.admin`?

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
-lxml==4.9.3
```
Comment (samuel-iseli) [inline requirements.txt:1]: why remove these packages?

**Review by samuel-iseli — COMMENTED** [review]
I would like to understand why the required sequence of `juntagrico_billing` and `juntagrico` in `settings.py` changed.

The other thing I don't understand is, why we need to register juntagrico admin views in juntagrico_billing.

Otherwise the changes seem OK to me. Didn't try to run it myself though...

File: README.md
Comment (pascalfree) [reply README.md:1]: This is new in juntagrico 1.6. It allows template overrides: https://juntagrico.readthedocs.io/en/release_1.6/reference/templates.html

File: juntagrico_billing/admin/__init__.py (hunk 1 above)
Comment (pascalfree) [reply juntagrico_billing/admin/__init__.py:31]: These are additional inlines that come from this addon.
They were in `juntagricoapps.py` before, which is now obsolete.

File: requirements.txt (hunk 2 above)
Comment (pascalfree) [reply requirements.txt:1]: They are moved to projectpy.toml and are installed locally with the `-e .` line.

File: README.md
Comment (samuel-iseli) [reply README.md:1]: OK, but this will be a breaking change for all installations that use the `juntagrico-billing` app.

**Review by samuel-iseli — APPROVED** [review]

**Pull request merged**